### PR TITLE
Release 0.13.31

### PR DESCRIPTION
**Pull request by gerolf-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,51 @@ Release notes
...
+Navigator
+~~~~~~~~~
+
+- Accept and use an access token to be used against Ledger API servers that require authentication, see `issue #3156 <https://github.com/digital-asset/daml/issues/3156>`_.
+- Demo-oriented password workflow has been removed.
+
+Ledger Client
+~~~~~~~~~~~~~
+
+- Expose new method to construct channels for more granular control overy the client creation process.
```
Comment (stefanobaghino-da) [inline docs/source/support/release-notes.rst]:
```suggestion
- Expose new method to construct channels for more granular control over the client creation process.
```
My fault, sorry.

**Comment by gerolf-da** [comment]
🎉  All manual tests have passed 🎉

**Pull request merged**